src/csharp/Extensions/Microsoft.Spark.Extensions.Delta.E2ETest/DeltaTableTests.cs

```diff
@@ -326,6 +327,22 @@ public void TestSignatures()
                 })));
         }
 
+        /// <summary>
+        /// Test that Delta Lake 0.7+ methods return the expected signature.
+        /// </summary>
+        [SkipIfSparkVersionIsLessThan(Versions.V3_0_0)]
+        public void TestSpark3Signatures()
```

Comment:
Should we use the naming convention used in the Microsoft.Spark tests.  The one above this we should name `public void TestSignaturesV2_3_X()` maybe ?

```suggestion
        public void TestSignaturesV3_0_X()
```

Reply:
Fixed.